### DISCUSSION TITLE
How to add exceptions to the Annoyances Filter?

### DISCUSSION THREAD

**Discussion by bitsper2nd** [DandelionSprout/adfilt#187] [Q&A] [answered]
Hi, I'm curious if it is possible to your Annoyances filter, but add exceptions to it. I like reading and laughing at comments on youtube and ghacks, so excluding the antiawfulcomments list is a must for me.

**Comment by DandelionSprout** [comment, marked as answer]
You can try with these for the immediate time being:
```
ghacks.net,youtube.com#@##comments
youtube.com#@##watch-discussion
youtube.com#@#ytd-live-chat-frame
@@||youtube.com/live_chat_fragments_ajax?$xmlhttprequest
@@||youtube.com/comment_service_ajax?$domain=youtube.com
```

You have given me an additional idea that could become a game-changer in the long term (i.e. some months from now), but it'll most likely only work in uBO and not AdGuard.

**Reply by DandelionSprout** [reply]
https://github.com/uBlockOrigin/uBlock-issues/issues/1599

**Reply by bitsper2nd** [reply]
Thx for the quick fix! BTW have you tried Sublime Text 4? It feels smoother than version 3.

**Reply by DandelionSprout** [reply]
I began using it a few days ago. Apart from how I reverted back from the Marina theme to 3's stock theme, I like it so far.

**Reply by bitsper2nd** [reply]
Same. I really didn't mind the price increase. Been using it for almost 5 years. Apart from better autocomplete, they also added this new [feat](https://www.sublimetext.com/docs/tab_multi-select.html). Odatnurd's [video](https://www.youtube.com/watch?v=3wUBrW3bsoo) overview convinced me to try the new version.

**Reply by JsfasdF252** [reply]
@Who what when where
